src/EventHandler/Events.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\EventHandler;
+
+interface Events
+{
+    public const START = 'StartSpanEvent';
```

Comment:
needs to be more specific, probably `SPAN_START` and `SPAN_END` - other things can start and end, and we might want to have listeners on those events, too, in the future.

Reply:
Yes the SPAN_START makes sense. 'StartSpanEvent' is the value which is to be returned by getClassName